src/client/pages/OfferNew/Introduction/index.tsx

```diff
@@ -1,56 +1,74 @@
+import React, { useState } from 'react'
 import styled from '@emotion/styled'
 import { colorsV3 } from '@hedviginsurance/brand'
-import React from 'react'
-import { Container, Section } from 'pages/OfferNew/components'
+import { Section } from 'pages/OfferNew/components'
 import { OfferData } from 'pages/OfferNew/types'
-import { isBundle } from 'pages/OfferNew/utils'
-import { useDocumentScroll } from '../../../utils/hooks/useDocumentScroll'
-import { ExternalInsuranceProvider } from './ExternalInsuranceProvider'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { HeroOfferDetails } from './HeroOfferDetails'
 import { Sidebar } from './Sidebar'
 
-import { Usps } from './USPS'
-
-interface Props {
+type Props = {
   offerData: OfferData
   refetch: () => Promise<void>
   onCheckoutOpen: () => void
 }
 
+type HeroImageProps = {
+  hasLoaded: boolean
+}
+
+const HERO_HEIGHT = '400px'
+
 const Wrapper = styled.div`
   width: 100%;
-  padding: 7rem 0;
-  background-color: ${colorsV3.gray900};
-  position: relative;
-  box-sizing: border-box;
+  height: ${HERO_HEIGHT};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`
 
-  @media (max-width: 1020px) {
-    padding: 7rem 0 7rem 0;
-  }
+const Hero = styled.div`
+  width: 100vw;
+  height: ${HERO_HEIGHT};
+  background: ${colorsV3.gray900};
+  overflow: hidden;
+  position: absolute;
+`
+
+const HeroImage = styled.img<HeroImageProps>`
+  height: 1000px;
+  object-fit: cover;
+  object-position: -550px -100px;
+  opacity: ${({ hasLoaded }) => (hasLoaded ? 0.5 : 0)};
+  transition: opacity 0.8s;
 
-  @media (max-width: 600px) {
-    padding: 7rem 0 5rem 0;
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    object-position: right -88px;
```

Comment:
Ah! Nice catch - I don't have  a monitor at home so couldn't easily check on a very large screen. So this 👇  will fix it?
```javascript
 ${LARGE_SCREEN_MEDIA_QUERY} {
    height: auto;
    object-position: right -5vw;
```

Reply:
No worries, hard to notice without it! Yes indeed 🙌 And this might be one of the few times safari might be of use :D you can check with their "Responsive design mode"
<img width="1487" alt="Screenshot 2021-01-12 at 16 43 11" src="https://user-images.githubusercontent.com/6661511/104337175-7adda980-54f5-11eb-8a3d-bb2c6e4c83cc.png">

Reply:
What??!! I had no idea 🤯  Thank you! 👑